config/settings.yml

```diff
@@ -62,6 +62,7 @@ ihub:
   appointments:
     timeout: 30
     mock: false
+  in_production: false
```

Comment:
I think this is temporary during development right? Maybe a comment to remind us to change it?

Reply:
No, actually, that would remain.  The way iHub has structured this endpoint is that parameter must be passed in staging, and must not be passed in production.

Reply:
I might be confused. The settings.yml file has the default production settings, right? Is it weird to have `in_production: false`... in production?